### PR TITLE
cuTENSOR v1.3 on documentation

### PR DESCRIPTION
**Pull request by takagi**
Blocked by chainer/chainer-test#655 and https://github.com/cupy/cupy/pull/5192

### CODE REVIEW

**Review by leofang — APPROVED** [review]
LGTM. I think we can backport this PR, with an additional note that to use cuTENSOR v1.3.0 with CuPy v9, users need to build CuPy from source.

File: docs/source/install.rst [hunk 1]
```diff
@@ -50,7 +50,7 @@ Additional CUDA Libraries
 
 Part of the CUDA features in CuPy will be activated only when the corresponding libraries are installed.
 
-* `cuTENSOR <https://developer.nvidia.com/cutensor>`_: v1.2 (CUDA 10.1+)
+* `cuTENSOR <https://developer.nvidia.com/cutensor>`_: v1.2 (CUDA 10.1) / v1.3 (CUDA 10.2+)
```
Comment (kmaehashi) [inline docs/source/install.rst:53]: Let's drop support for cuTENSOR 1.2 in v10 series.

```suggestion
* `cuTENSOR <https://developer.nvidia.com/cutensor>`_: v1.3 (CUDA 10.2+)
```

File: docs/source/install.rst [hunk 2]
```diff
@@ -50,7 +50,7 @@ Additional CUDA Libraries
 
 Part of the CUDA features in CuPy will be activated only when the corresponding libraries are installed.
 
-* `cuTENSOR <https://developer.nvidia.com/cutensor>`_: v1.2 (CUDA 10.1+)
+* `cuTENSOR <https://developer.nvidia.com/cutensor>`_: v1.3 (CUDA 10.2+)
```
Comment (kmaehashi) [inline docs/source/install.rst:53]:
```suggestion
* `cuTENSOR <https://developer.nvidia.com/cutensor>`_: v1.3
```

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Comment by kmaehashi** [comment]
When backporting add v1.2 to versions supported.

**Pull request merged**